drivers/ds3231/ds3231.c

```diff
@@ -236,6 +239,140 @@ int ds3231_set_time(const ds3231_t *dev, const struct tm *time)
     return 0;
 }
 
+int ds3231_set_alarm_1(const ds3231_t *dev, struct tm *time,
+                       ds3231_alm_1_mode_t trigger)
+{
+    uint8_t raw[A1_REG_NUMOF];
+    uint8_t a1mxMask[A1_REG_NUMOF];
+
+    /* A1M1, A1M2, A1M3 and A1M4 are set accordingly to the trigger type */
+    a1mxMask[0] = (trigger & 0x01) << 7;
```

Comment:
Please use lowercase name for variables, e.g.: a1mx_mask 😉

Reply:
Yes sorry, force of habit :grimacing: